odoo/custom/dependencies/pip.txt

```diff
@@ -23,3 +23,11 @@ pydantic
 pyquerystring
 typing-extensions
 schwifty
+
+# OpenSPP-API dependencies
+bravado_core
+jsonschema
+jwt
+PyLD
+swagger_spec_validator
+cryptography<38
```

Comment:
Self-comment: This prevents OpenSPP JWT Rest API Authentication from being installed, as it requires pyOpenSSL 22.0.0 (or higher) to be installed.

@reichie020212 @jeremi we need to resolve this by adjusting the code in `openspp_api` to support the latest version of pyOpenSSL.

Reply:
I will remove the pinning of `cryptography` now.